docs/MO_DG/prepare_model/FP16_Compression.md

```diff
@@ -2,12 +2,12 @@
 
 @sphinxdirective
 
-Model Optimizer can convert all floating-point weights to the ``FP16`` data type. 
+Model Conversion API can convert all floating-point weights to the ``FP16`` data type. 
```

Comment:
Please do not use `Model Conversion API` everywhere because the tool will still exist. So just rename everywhere `Model Optimizer` to `MO`